.github/workflows/test.yaml

```diff
@@ -65,4 +65,4 @@ jobs:
       run: sudo apt-get install xclip
 
     - name: Run e2e tests
-      run: xvfb-run pytest --e2e --log-cli-level=INFO test/e2e
```

Comment:
Can you keep the `test/e2e` part here? It makes sure only those tests run in this workflow and keeps the output a bit prettier (doesn't show skipped testcases).

Reply:
done